egov/egov-collectionweb/src/main/java/org/egov/collection/web/actions/receipts/BankRemittanceAction.java

```diff
@@ -266,6 +296,34 @@ public String create() {
         totalCashAmount = getSum(finalList);
         return INDEX;
     }
+    
+    public List<ReceiptBean> prepareBankRemittanceList() {
+        finalList = new ArrayList<>();
+        String[] selectedRowsIdArray;
+        if (selectedRowsId != null)
+            selectedRowsIdArray = selectedRowsId.split(";,");
+        else
+            selectedRowsIdArray = new String[0];
+        int length = selectedRowsIdArray.length;
```

Comment:
don't take the length into a local variable. Use it directly in the place where the length local variable is used.